### PR TITLE
Updated naming

### LINKED ISSUE #205: Update naming in dapp to match design naming
## Description

Currently, a lot of pages in the dapp follow a different naming scheme than the ones in the proposed design. This can create a lot of confusion. We should follow the same naming scheme in both the design and code to avoid confusion.

@roy-castrillo We need to spend some time together for this to set a terminology that we both follow afterward.

## Acceptance criteria
-  Consistent naming of screens across design and implementation

## Tasks
- [x] Check for inconsistencies in the naming between design and implementation
- [x] Decide on naming
- [ ] Update naming on implementation

### PR DESCRIPTION
**Pull request by kelsos**
Closes #205

### CODE REVIEW

File: raiden-dapp/src/components/Tokens.vue [hunk 1]
```diff
@@ -48,7 +48,7 @@
               <v-layout justify-center row>
                 <v-btn
                   :id="`pay-${index}`"
-                  :to="`/transfer/${token.address}`"
+                  :to="`/send-transaction/${token.address}`"
```
Comment (andrevmatos) [inline raiden-dapp/src/components/Tokens.vue]: I'd say it shouldn't be called `send-transaction`, as transaction remembers on-chain transactions (txs), while payments/transfers on Raiden are fully off-chain and aren't related to actual transactions. Raiden Python calls these just payments, in the sence that, in the future, a single payment could be split in multiple transfers. We should stick with this terminology here.

File: raiden-dapp/src/mixins/navigation-mixin.ts [hunk 2]
```diff
@@ -30,7 +30,7 @@ export default class NavigationMixin extends Vue {
 
   navigateToSelectPaymentTarget(token: string) {
     this.$router.push({
-      name: RouteNames.TRANSFER,
+      name: RouteNames.SEND_TRANSACTION,
```
Comment (andrevmatos) [inline raiden-dapp/src/mixins/navigation-mixin.ts]: Same

File: raiden-dapp/src/mixins/navigation-mixin.ts [hunk 3]
```diff
@@ -43,7 +43,7 @@ export default class NavigationMixin extends Vue {
 
   onBackClicked() {
     switch (this.$route.name) {
-      case RouteNames.TRANSFER:
+      case RouteNames.SEND_TRANSACTION:
```
Comment (andrevmatos) [inline raiden-dapp/src/mixins/navigation-mixin.ts]: Same

File: raiden-dapp/src/route-names.ts [hunk 4]
```diff
@@ -1,5 +1,5 @@
 export enum RouteNames {
-  TRANSFER = 'transfer',
+  SEND_TRANSACTION = 'send-transaction',
```
Comment (andrevmatos) [inline raiden-dapp/src/route-names.ts]: ³

File: raiden-dapp/src/router.ts [hunk 5]
```diff
@@ -23,12 +23,12 @@ export default new Router({
       redirect: '/'
     },
     {
-      path: '/transfer/:token',
-      name: RouteNames.TRANSFER,
+      path: '/send-transaction/:token',
+      name: RouteNames.SEND_TRANSACTION,
```
Comment (andrevmatos) [inline raiden-dapp/src/router.ts]: ^

File: raiden-dapp/src/views/SendTransaction.vue [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+    ActionButton,
+    TokenInformation,
+    Divider,
+    AddressInput,
+    AmountInput,
+    Stepper,
+    ErrorScreen
+  }
+})
+export default class SendTransaction extends Vue {
```
Comment (andrevmatos) [inline raiden-dapp/src/views/SendTransaction.vue]: ^

File: raiden-dapp/src/components/Tokens.vue (hunk 1 above)
Comment (kelsos) [reply raiden-dapp/src/components/Tokens.vue]: Yes, it makes more sense to follow the same semantics and terminology as the python documentation. We will rename it to payment then for consistency.

**Pull request merged**